Dockerfile.pr-builder

```diff
@@ -0,0 +1,60 @@
+FROM continuumio/miniconda
+MAINTAINER Hail Team <hail@broadinstitute.org>
+
+USER root
+RUN apt-get -y install \
+    bash \
+    cmake \
+    curl \
+    g++ \
+    git \
+    openjdk-8-jdk-headless \
+    r-base \
+    tar \
+    unzip
+
+RUN apt-get -y install libnlopt-dev # https://github.com/jyypma/nloptr/issues/40
+RUN Rscript -e 'install.packages(c("jsonlite", "SKAT", "logistf"), repos = "http://cran.us.r-project.org")'
+RUN apt-get -y install r-cran-ncdf4 r-cran-rnetcdf libcurl4-openssl-dev libxml2-dev
+RUN Rscript -e 'source("https://bioconductor.org/biocLite.R"); biocLite("GENESIS"); biocLite("SNPRelate"); biocLite("GWASTools")'
```

Comment:
From what I've read, it seems suggested best practice is to minimize the number of layers in your image. Each of these `RUN` commands adds a layer. I think it would be good to group things together as much as possible, keeping separate layers only when we want the change insulation (changing a later layer doesn't require rebuilding previous layers).